src/tests/ie_test_utils/common_test_utils/graph_comparator.cpp

```diff
@@ -504,6 +504,17 @@ class CompareSubGraphs {
             }
         }
 
+        const auto lhs_body = sub_lhs->get_function();
```

Comment:
I think we need to create a ticket to extend this approach for n functions (e.g. If operation case)

Reply:
It would require extension of comparator to handle multi subgraph nodes at all, what should be a separate task.

Reply:
yes, please just create a new jira ticket for this

Reply:
Created CVS-114841